packages/protocol/contracts/common/AddressResolver.sol

```diff
@@ -25,7 +26,7 @@ import "./AddressManager.sol";
 /// Note that the address manager should be changed using upgradability, there
 /// is no setAddressManager() function go guarantee atomicness across all
 /// contracts that are resolvers.
-abstract contract AddressResolver {
+abstract contract AddressResolver is Initializable {
```

Comment:
if AddressResolver is Initializable, then EssentialContract is Initializable by default.. 
So you can delete "is Initializable, " for EssentialContract, and all its sub contracts.

Reply:
Understood. I've observed similar initialization flows in OpenZeppelin contracts. While the parent contracts already imported Initializable, the child contracts redundantly re-imported it for code cleanliness. Following your guidance, I've removed these redundant imports.